Mathematics/Roman numerals/leetcode/13. Roman to Integer/sonnet 5/Roman_to_Integer_Python.md

```
@@ -0,0 +1,286 @@
+# Roman to Integer（ローマ数字を整数に変換する）
+
+---
+
+### 1. 問題分析結果
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)